SPECS/python-mock/python-mock.signatures.json

```diff
@@ -1,5 +1,5 @@
 {
- "Signatures": {
-  "mock-3.0.5.tar.gz": "ff286cec703de8770f735269575ecb7d1c937b8851eba8de94d800df084e627c"
- }
+  "Signatures": {
```

Comment:
Nit: Can we go back to the previous indentation here?

Reply:
Sure. The script I used needs a small update to make it stop changing that.